### PR TITLE
Add surgeries for healing Organs

### PR DESCRIPTION
**Pull request by Bod9001**
and improved surgery description for the current ones
and fixes bandages and ointment

### Changelog:
CL:Add surgeries for healing Organs

### CODE REVIEW

File: UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset [hunk 1]
```diff
@@ -12,15 +12,15 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 99f67d4c0b7de7545800f4fddbcc58cd, type: 3}
   m_Name: GenericCloseProcedure
   m_EditorClassIdentifier: 
-  ProcedureName: 'Cclose '
+  ProcedureName: Cose
```
Comment (corp-0) [inline UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset]: typo

Comment (Bod9001) [reply UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset]: typo to typo
 xD

File: UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset [hunk 2]
```diff
@@ -12,15 +12,15 @@ MonoBehaviour:
...
+    Time: 1
     SuccessChance: 100
-    StartSelf: 
-    StartOther: 
-    SuccessSelf: 
-    SuccessOther: 
-    FailSelf: 
-    FailOther: 
+    StartSelf: you start to use {Using} to seal up {WhoOn}'s {OnPart}
+    StartOther: '{performer} uses {Using} starts to close up {WhoOn}''s {OnPart}'
```
Comment (corp-0) [inline UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset]:
```suggestion
    StartOther: '{performer} uses {Using} and starts to close up {WhoOn}''s {OnPart}'
```
Maybe? Feel like a word is missing  there.

Comment (Bod9001) [reply UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset]: Using is the tool name

Comment (Bod9001) [reply UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset]: probably a "," would work Better

File: UnityProject/Assets/ScriptableObjects/Surgery/GenericImplantProcedure.asset [hunk 3]
```diff
@@ -18,38 +18,38 @@ MonoBehaviour:
...
     Time: 1
     SuccessChance: 100
-    StartSelf: 
-    StartOther: 
-    SuccessSelf: 
-    SuccessOther: 
-    FailSelf: 
-    FailOther: 
+    StartSelf: you start to use {Using} to cut open {WhoOn}'s {OnPart}
+    StartOther: '{performer} starts to use {Using} to cut open {WhoOn}''s {OnPart}'
```
Comment (corp-0) [inline UnityProject/Assets/ScriptableObjects/Surgery/GenericImplantProcedure.asset:22]:
```suggestion
    StartOther: '{performer} starts to use {Using} to cut open {WhoOn}'s {OnPart}'
```

File: UnityProject/Assets/ScriptableObjects/Surgery/GenericImplantProcedure.asset [hunk 4]
```diff
@@ -18,38 +18,38 @@ MonoBehaviour:
...
+    SuccessSelf: you use {Using} to cut open {WhoOn}'s {OnPart}
+    SuccessOther: '{performer} uses {Using} to cut open {WhoOn}''s {OnPart}'
```
Comment (corp-0) [inline UnityProject/Assets/ScriptableObjects/Surgery/GenericImplantProcedure.asset:24]:
```suggestion
    SuccessOther: '{performer} uses {Using} to cut open {WhoOn}'s {OnPart}'
```

File: UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset (hunk 2 above)
Comment (Bod9001) [reply UnityProject/Assets/ScriptableObjects/Surgery/GenericCloseProcedure.asset]: oh starting

**Comment by corp-0** [comment]
is "You miss cutts" grammathically correct?

**Comment by Bod9001** [comment]
it should be "you miss cut"

**Comment by MaxIsJoe** [comment]
> it should be "you miss cut"

No, it's "miscut"
confirmed [here](https://www.merriam-webster.com/dictionary/miscut)

**Comment by Bod9001** [comment]
the extra " ' " is formatting on The file

File: UnityProject/Assets/ScriptableObjects/Surgery/GenericImplantProcedure.asset [hunk 5]
```diff
@@ -18,38 +18,38 @@ MonoBehaviour:
...
+    FailSelf: you miss cutts with {Using} cutting into{WhoOn}'s {OnPart}
```
Comment (corp-0) [inline UnityProject/Assets/ScriptableObjects/Surgery/GenericImplantProcedure.asset]:
```suggestion
    FailSelf: you miss cut with {Using} cutting into {WhoOn}'s {OnPart}
```

File: UnityProject/Assets/ScriptableObjects/Surgery/GenericImplantProcedure.asset [hunk 6]
```diff
@@ -18,38 +18,38 @@ MonoBehaviour:
...
-    StartOther: 
-    SuccessSelf: 
-    SuccessOther: 
-    FailSelf: 
-    FailOther: 
+    StartSelf: you try to insert {Using} into {WhoOn}'s {OnPart}
+    StartOther: '{performer} tries to insert {Using} into {WhoOn}''s {OnPart}'
+    SuccessSelf: you successfully insert {Using} into {WhoOn}'s {OnPart}
+    SuccessOther: '{performer} successfully inserts {Using} into {WhoOn}''s {OnPart}'
+    FailSelf: you fiail to fit {Using} into {WhoOn}'s {OnPart}
```
Comment (corp-0) [inline UnityProject/Assets/ScriptableObjects/Surgery/GenericImplantProcedure.asset]:
```suggestion
    FailSelf: you fail to fit {Using} into {WhoOn}'s {OnPart}
```

File: UnityProject/Assets/ScriptableObjects/Surgery/GenericRemoveProcedure.asset [hunk 7]
```diff
@@ -18,27 +18,27 @@ MonoBehaviour:
...
-    StartOther: 
-    SuccessSelf: 
-    SuccessOther: 
-    FailSelf: 
-    FailOther: 
+    StartSelf: you start to use {Using} to cut open {WhoOn}'s {OnPart}
+    StartOther: '{performer} starts to use {Using} to cut open {WhoOn}''s {OnPart}'
+    SuccessSelf: you use {Using} to cut open {WhoOn}'s {OnPart}
+    SuccessOther: '{performer} uses {Using} to cut open {WhoOn}''s {OnPart}'
+    FailSelf: you miss cutts with {Using} cutting into{WhoOn}'s {OnPart}
```
Comment (corp-0) [inline UnityProject/Assets/ScriptableObjects/Surgery/GenericRemoveProcedure.asset]:
```suggestion
    FailSelf: you miss cut with {Using} cutting into {WhoOn}'s {OnPart}
```

**Comment by PerfectTangent** [comment]
> > it should be "you miss cut"
> 
> No, it's "miscut"
> confirmed [here](https://www.merriam-webster.com/dictionary/miscut)

Either spelling is perfectly understandable. I'd personal go with miscut in this situation, though.

**Comment by Cboyjet4** [comment]
> > > it should be "you miss cut"
> > 
> > 
> > No, it's "miscut"
> > confirmed [here](https://www.merriam-webster.com/dictionary/miscut)
> 
> Either spelling is perfectly understandable. I'd personal go with miscut in this situation, though.

A possible other could be "You cut _____ in the wrong place! it makes more sense.
Alternative "You misscut _____ in the wrong place!"

**Pull request merged**